### PR TITLE
Fix #15588: Ctrl+Up/Down do not change tablature strings in note input mode

### LINKED ISSUE #15588: [MU4 Issue] Ctrl+Up/Down do not change tablature strings in note input mode
**Describe the bug**
Ctrl+Up/Down work to change strings in normal mode but not in note input mode

**To Reproduce**
1. Create guitar tablature score 
2. Enter a note on top string, fret 0
3. While still in note input mode, press Ctrl+Down
4. See error: cursor moves to next string, but note stays on top string

**Expected behavior**
The fret 0 top string should change to fret 5 next string, as it does in normal mode

**Platform information**
 - OS: Linux

**Additional context**
This worked in MU3

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #15588 

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/notation/internal/notationactioncontroller.cpp [hunk 1]
```diff
@@ -932,14 +932,22 @@ void NotationActionController::move(MoveDirection direction, bool quickly)
...
         } else {
+            bool isTab = interaction->noteInput()->isNoteInputMode()
+                         && interaction->noteInput()->state().staffGroup == mu::engraving::StaffGroup::TAB;
+            int ostring = isTab && selectedElement->isNote() ? toNote(selectedElement)->string() : -1;
+
             interaction->movePitch(direction, quickly ? PitchMode::OCTAVE : PitchMode::CHROMATIC);
+
+            if (isTab && selectedElement->isNote()) {
+                int nstring = toNote(selectedElement)->string();
+                if (ostring != nstring) {
```
Comment (Eism) [inline src/notation/internal/notationactioncontroller.cpp:946]: Please rename 
`ostring` to `oldString` 
`nstring` to `newString`

And please add new variable `isNote = isTab && selectedElement->isNote()` and reuse

**Comment by DmitryArefiev** [comment]
@HemantAntony Can you rebase please?

**Comment by HemantAntony** [comment]
@DmitryArefiev rebased :)

**Comment by DmitryArefiev** [comment]
Last test results in [comment](https://github.com/musescore/MuseScore/issues/15588#issuecomment-1584656831)

**Comment by RomanPudashkin** [comment]
Closing, the original issue was fixed in 4.2 (confirmed by @DmitryArefiev)

**Pull request closed without merging**